app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,181 @@
+<form class="ui form" autocomplete="off" {{action 'submit' on='submit' }}>
```

Comment:
`order-form.hbs` is missing translation helper, please add it for all the strings.